src/themes/default/components/core/Logo.vue

```diff
@@ -4,13 +4,22 @@
       :width="width"
       :height="height"
       src="/assets/logo.svg"
-      alt="Vuestore logo"
+      :alt="$t(defaultTitle)"
     >
   </router-link>
 </template>
 
 <script>
+import config from 'config'
+import { currentStoreView } from '@vue-storefront/core/lib/multistore'
+
 export default {
+  data () {
+    const storeView = currentStoreView()
+    return {
+      defaultTitle: storeView.seo.defaultTitle ? storeView.seo.defaultTitle : 'Vuestore'
```

Comment:
we always have this in default config
```suggestion
      defaultTitle: storeView.seo.defaultTitle ? storeView.seo.defaultTitle : config.seo.defaultTitle
```

Reply:
- Added changes